src/main/resources/validationrules/employment/employment-judicial-office-holder-mapping.drl

```diff
@@ -0,0 +1,209 @@
+                               (endTime == null || endTime.compareTo(ZonedDateTime.now()) >= 0),
+                               (validateAuthorisation(authorisations, "BHA1") ))
+then
+  insert(
+      JudicialOfficeHolder.builder()
+      .userId($jap.getUserId())
+      .office("EMPLOYMENT Employment Judge-Salaried")
+      .jurisdiction("EMPLOYMENT")
+      .ticketCodes($jap.getTicketCodes())
+      .beginTime($jap.getBeginTime())
+      .endTime($jap.getEndTime())
+      .regionId($jap.getRegionId())
+      .contractType($jap.getAppointmentType())
+      .build());
+      logMsg("Rule : employment_employment_judge_salaried_joh");
+end;
+
+/*
+ * "EMPLOYMENT Employment Judge" business role mapping to JOH.
+ */
+rule "employment_employment_judge_fee_paid_joh"
+when
+   $f:  FeatureFlag(status && flagName == FeatureFlagEnum.EMPLOYMENT_WA_1_0.getValue())
+   $jap: JudicialAccessProfile(appointment != null,
+                               appointment == "Employment Judge",
+                               appointmentType == "Fee-Paid",
+                               (endTime == null || endTime.compareTo(ZonedDateTime.now()) >= 0),
+                               (validateAuthorisation(authorisations, "BHA1") ))
+then
+  insert(
+      JudicialOfficeHolder.builder()
+      .userId($jap.getUserId())
+      .office("EMPLOYMENT Employment Judge-Fee-Paid")
+      .jurisdiction("EMPLOYMENT")
+      .ticketCodes($jap.getTicketCodes())
+      .beginTime($jap.getBeginTime())
+      .endTime($jap.getEndTime())
+      .regionId($jap.getRegionId())
+      .contractType($jap.getAppointmentType())
+      .build());
+      logMsg("Rule : employment_employment_judge_fee_paid_joh");
+end;
+
+/*
+ * "EMPLOYMENT Tribunal Member" business role mapping to JOH.
+ */
+rule "employment_tribunal_member_fee_paid_joh"
+when
+   $f:  FeatureFlag(status && flagName == FeatureFlagEnum.EMPLOYMENT_WA_1_0.getValue())
+   $jap: JudicialAccessProfile(appointment != null,
+                               appointment == "Tribunal Member",
+                               appointmentType == "Fee-Paid",
+                               (endTime == null || endTime.compareTo(ZonedDateTime.now()) >= 0),
+                               (validateAuthorisation(authorisations, "BHA1") ))
+then
+  insert(
+      JudicialOfficeHolder.builder()
+      .userId($jap.getUserId())
+      .office("EMPLOYMENT Tribunal Member-Fee-Paid")
+      .jurisdiction("EMPLOYMENT")
+      .ticketCodes($jap.getTicketCodes())
+      .beginTime($jap.getBeginTime())
+      .endTime($jap.getEndTime())
+      .regionId($jap.getRegionId())
+      .contractType($jap.getAppointmentType())
+      .build());
+      logMsg("Rule : employment_tribunal_member_fee_paid_joh");
+end;
+
+/*
+ * "EMPLOYMENT Tribunal Member Lay" business role mapping to JOH.
+ */
+rule "employment_tribunal_member_lay_fee_paid_joh"
+when
+   $f:  FeatureFlag(status && flagName == FeatureFlagEnum.EMPLOYMENT_WA_1_0.getValue())
```

Comment:
As per the confluence an additional condition is required to give JOH, that is baseLocationId is of of  "1036" and "1037"